core/src/test/java/org/axonframework/config/ConfigAssertions.java

```diff
@@ -36,26 +37,16 @@ private ConfigAssertions() {
      * Asserts whether given {@code modules} are present in the provided {@code configuration}.
      *
      * @param configuration Axon configuration
-     * @param modules       Module configurations
+     * @param moduleTypes   expected module configuration types present in {@code configuration}
      */
     @SafeVarargs
     public static void assertExpectedModules(Configuration configuration,
-                                             Class<? extends ModuleConfiguration>... modules) {
-        List<ModuleConfiguration> configurationModules = configuration.getModules();
-        boolean[] matches = new boolean[configurationModules.size()];
-        for (Class<? extends ModuleConfiguration> module : modules) {
-            boolean found = false;
-            for (int i = 0; i < configurationModules.size(); i++) {
-                if (module.isInstance(configurationModules.get(i)) && !matches[i]) {
-                    found = true;
-                    matches[i] = true;
-                    break;
-                }
+                                             Class<? extends ModuleConfiguration>... moduleTypes) {
+        stream(moduleTypes).forEach(moduleType -> {
+            List<? extends ModuleConfiguration> matches = configuration.findModules(moduleType);
+            if (matches.isEmpty()) {
+                fail("No module of type '" + moduleType.getName() + "' was not found in the provided configuration");
```

Comment:
Typo: `...was found...` i.o. `...was *not* found...`

Reply:
Well spotted! 👍